attributes/default.rb

```diff
@@ -1,8 +1,8 @@
 #
-# Cookbook Name:: snort
+# Cookbook:: snort
 # Attributes:: default
 #
-# Copyright 2010-2015, Chef Software, Inc
+# Copyright:: 2010-2016, Chef Software, Inc
```

Comment:
So...I thought it was a yard thing, but it's not. I believe it's an rdoc thing that was adopted for cookbooks, but we never actually did anything about it. I've been setting them in a predictable way so we *could* potentially use them sometime in the future. I suspect what will probably happen is I'll convert this to a more yard like setup at some point.